common/transport/src/main/proto/grpc/block_master.proto

```diff
@@ -157,6 +157,11 @@ service BlockMasterClientService {
    */
   rpc GetWorkerInfoList(GetWorkerInfoListPOptions) returns (GetWorkerInfoListPResponse);
 
+  /**
+ * Returns a list of workers information.
+ */
+  rpc GetLostWorkerList(GetWorkerInfoListPOptions) returns (GetWorkerInfoListPResponse);
```

Comment:
this is also implemented in the master, right?

Reply:
Yes, it is. I just added a grpc method for client to call this method.